File: commands/cclist.py
import globals
import helpers
import discord

@globals.bot.command('cclist')
async def command_cclist(ctx):
	if f'{ctx.guild.id}' in globals.data[f'permitted servers']:
		desc = ''
		for key in globals.data[f'creation vcs']:
			desc += f'`{key}`\n'
		await helpers.sendEmbed(ctx, f'Creation channels:', desc, globals.defaultcolor, 20)

File: commands/cclist.js
module.exports = {
	name: "cclist",
	description: "List all creation channels",
	ephemeral: true,
	permissions: ["MANAGE_CHANNELS"],
	callback: (interaction, args, member, guild) => {
		let embed = new Discord.MessageEmbed()
			.setTitle('cclist command')
			.setDescription('List of creation channels')
			.setColor(global.defaultcolor);
		for (let k in global.data_creationVcs) {
			embed.addFields({name: k, value: global.data_creationVcs[k], inline: true});
		}
		return embed;
	},
	errorcallback: (err, interaction, args, member, guild) => {
		return new Discord.MessageEmbed().setTitle('Error').setDescription(err).setColor(global.errorcolor);
	}
};